recipe/meta.yaml

```diff
@@ -22,9 +22,15 @@ requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - cython                                 # [build_platform != target_platform]
```

Comment:
does it actually use cython and pybind11? these can have knock-on effects if not managed properly. i think if you check that blog post, you'll see the modifier "if needed"...

Reply:
@bollwyvl Okay, I removed those and it still works.